lib/ajv.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
```

Comment:
This is a potential candidate for a separate lib, but I would not bother for now.

Reply:
especially because package is aws-drivers